Mathlib/Topology/Category/Profinite/Basic.lean

```diff
@@ -250,4 +250,7 @@ theorem epi_iff_surjective {X Y : Profinite.{u}} (f : X ⟶ Y) : Epi f ↔ Funct
   · rw [← CategoryTheory.epi_iff_surjective]
     apply (forget Profinite).epi_of_epi_map
 
+/--The product space of profinite spaces is profinite-/
+def Pi.profinite {α : Type u} (β : α → Profinite) : Profinite := .of (Π (a : α), β a)
```

Comment:
Maybe here the name should just be `Profinite.pi`

Reply:
If you are sure it is better then I will change it, because I am not enough familiar with the naming convention.